lib/rules/no-tracked-properties-from-args.js

```diff
@@ -16,6 +16,7 @@ function visitClassPropertyOrPropertyDefinition(node, context, trackedImportName
   const hasThisArgsValue =
     node.value &&
     startsWithThisExpression(node.value) &&
+    nodeToDependentKey(node.value, context) &&
     nodeToDependentKey(node.value, context).split('.')[0] === 'args';
```

Comment:
Having to calculate the dependent key twice isn't ideal. Can't we use optional chaining here?

```suggestion
    nodeToDependentKey(node.value, context)?.split('.')[0] === 'args';
```

Reply:
Fair point!